platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/iss/IssDetector.java

```diff
@@ -462,4 +487,24 @@ private static void writeSkippedLogCount(final StringBuilder sb, final long skip
                     .append("seconds.");
         }
     }
+
+    /**
+     * @param n the notification to wrap
+     * @return a list containing the notification, or null if the notification is null
+     */
+    private List<IssNotification> returnList(@Nullable final IssNotification n) {
+        return n == null ? null : List.of(n);
+    }
+
+    /**
+     * @param list the list to filter
+     * @return the list, or null if the list is null or empty
+     */
+    private List<IssNotification> returnList(@Nullable final List<IssNotification> list) {
+        return list == null
+                ? null
+                : list.stream()
+                        .filter(Objects::nonNull)
+                        .collect(collectingAndThen(toList(), l -> l.isEmpty() ? null : l));
+    }
```

Comment:
Could this method be renamed to something more descriptive? E.g. `filterForSendableNotifications()`?

Reply:
renamed to `listOrNull`